### PR TITLE
Use browser language in UI

### PR DESCRIPTION
**Pull request by StefanKlt**
Set UI language according to browser settings. Fix unlocalized UI texts and refactor UI message IDs.

### CODE REVIEW

**Comment by schabdo** [comment]
@hawkbit-bot verify please

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/HawkbitCommonUtil.java [hunk 1]
```diff
@@ -561,4 +563,13 @@ public static String getArtifactoryDetailsLabelId(final String name, final Vaadi
         return getCaptionText(caption);
     }
 
+    public static Locale getLocaleToBeUsed(final UiProperties.Localization localizationProperties, final UI ui) {
+        final List<String> availableLocals = localizationProperties.getAvailableLocals();
+        final Locale uiLocale = ui.getSession().getLocale();
+        if (availableLocals.contains(uiLocale.getLanguage())) {
```
Comment (StefanKlt) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/HawkbitCommonUtil.java]: add comment how it works

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/TargetBulkUpdateWindowLayout.java [hunk 2]
```diff
@@ -336,9 +337,15 @@ public void onUploadCompletion() {
         minimizeButton.setEnabled(false);
     }
 
-    private static String getFormattedCountLabelValue(final int succussfulUploadCount, final int failedUploadCount) {
-        return new StringBuilder().append("Successful :").append(succussfulUploadCount)
-                .append("<font color=RED> Failed :").append(failedUploadCount).append("</font>").toString();
+    private String getFormattedCountLabelValue(final int succussfulUploadCount, final int failedUploadCount) {
```
Comment (bogdan-bondar) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/TargetBulkUpdateWindowLayout.java]: typos in _succussfulUploadCount_ parameter, and I would rename the _ stringBuilder_ local variable to e.g. _countLabelBuilder_

**Comment by schabdo** [comment]
@hawkbit-bot verify please

**Pull request merged**